### ISSUE TITLE
pytest ordering warning in tests

### ISSUE THREAD

**Issue by jni** [napari/napari#1243]
## 🐛 Bug

Just started noticing this warning when running tests:

```python
napari/_tests/test_viewer.py:50
  /home/jni/projects/napari/napari/_tests/test_viewer.py:50: PytestUnknownMarkWarning: Unknown pytest.mark.first - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    @pytest.mark.first  # provided by pytest-ordering
```

Anyone else notice this?

**Comment by tlambert03** [comment]
I feel like I have in the past... but I think i somehow didn't have `pytest-ordering` in my env... you double checked that you have it?

**Comment by jni** [comment]
I did. Also, back when I first implemented this test, the entire suite failed without that ordering, that's why it's in there! :question: :question: :question:

**Comment by sofroniewn** [comment]
Closed by #1383
